ajax/libs/iphone-inline-video/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "iphone-inline-video",
+  "filename": "iphone-inline-video.min.js",
+  "description": "Make videos playable inline on the iPhone (prevents automatic fullscreen)",
+  "homepage": "https://bfred-it.github.io/iphone-inline-video/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bfred-it/iphone-inline-video.git"
+  },
+  "author": "Federico Brigante <github@bfred.it> (bfred.it)",
+  "keywords": [
+    "browser",
+    "iphone",
+    "ios",
+    "play",
+    "video",
+    "inline",
+    "fullscreen",
+    "full screen",
+    "ipod",
+    "autoplay",
+    "multiple",
+    "audio",
+    "sound",
+    "canvas",
+    "gif"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bfred-it/iphone-inline-video.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
```

Comment:
Could we use npm auto-update instead, please?

https://www.npmjs.com/package/iphone-inline-video

```
tree node_modules/iphone-inline-video

node_modules/iphone-inline-video
├── CHANGELOG.md
├── LICENSE
├── README.md
├── dist
│   ├── iphone-inline-video.common-js.js
│   ├── iphone-inline-video.es-modules.js
│   ├── iphone-inline-video.js
│   └── iphone-inline-video.min.js
└── package.json

1 directory, 8 files
```